tests/cupyx_tests/scipy_tests/signal_tests/test_signaltools.py

```diff
@@ -55,6 +55,59 @@ def test_correlate_float16(self, xp, scp, dtype=cupy.float16):
         return self._filter('correlate', dtype, xp, scp)
 
 
+@testing.parameterize(*testing.product({
+    'in1': [(10,), (5, 10), (10, 3), (3, 4, 10)],
```

Comment:
How about renaming to `size1`? In `_filter` method, `self.in1` is an integer that represents the input size, and `in1` is `xp.ndarray`.